src/BizHawk.Client.Common/movie/tasproj/ZwinderStateManager.cs

```diff
@@ -458,7 +469,7 @@ public bool InvalidateAfter(int frame)
 			var b1 = InvalidateNormal(frame);
 			var b2 = InvalidateGaps(frame);
 			var b3 = InvalidateReserved(frame);
-			StateCache.RemoveWhere(s => s > frame);
+			StateCache.RemoveAll(s => s > frame);

```

Comment:
Seems like this could be replaced by a `RemoveAfter` method on the sorted list, which would have better runtime characteristics.

But if this is fast enough, maybe we don't care more.

Reply:
Done in e050938.